### PR TITLE
PS-1368: Feat(): badge and larger input and upload

### PR DESCRIPTION
**Pull request by adlenafane**
https://github.com/alkemics/dashboard-flux/pull/1048
https://github.com/alkemics/sdk-dashboard/pull/287
https://github.com/alkemics/alk-style-common/pull/88
https://github.com/alkemics/alk-ng-component/pull/63

### CODE REVIEW

File: src/material-ui/_inputs.scss [hunk 1]
```diff
@@ -98,20 +98,22 @@ select, .input-group select, .input-group select.form-control, .form-inline sele
...
-        float: none;
-        border: 0;
-        box-shadow: none;
-        border-radius: 0;
-        background: transparent;
-        border-bottom: 1px solid $input-border-color;
-        &:disabled {
-            border-style: dashed;
+body {
+    fieldset[disabled] .form-control, .form-control-wrapper .form-control, .form-control {
```
Comment (birdy-) [inline src/material-ui/_inputs.scss]: Seems very generic ?

**Comment by birdy-** [comment]
small comment otherwiser LGTM

File: src/material-ui/_inputs.scss (hunk 1 above)
Comment (adlenafane) [inline src/material-ui/_inputs.scss]: @birdy- What do you mean?

**Comment by NFau** [comment]
LGTM

**Pull request merged**